common/build-helper/python3.sh

```diff
@@ -0,0 +1,14 @@
+# fix building non-pure-python modules on cross
+if [ -n "$CROSS_BUILD" ]; then
+	export PYPREFIX="$XBPS_CROSS_BASE"
+	export CFLAGS+=" -I${XBPS_CROSS_BASE}/${py3_inc} -I${XBPS_CROSS_BASE}/usr/include"
+	export LDFLAGS+=" -L${XBPS_CROSS_BASE}/${py3_lib} -L${XBPS_CROSS_BASE}/usr/lib"
+	export CC="${XBPS_CROSS_TRIPLET}-gcc -pthread $CFLAGS $LDFLAGS"
+	export LDSHARED="${CC} -shared $LDFLAGS"
+	export PYTHONPATH="${XBPS_CROSS_BASE}/${py3_lib}"
+	for f in ${XBPS_CROSS_BASE}/${py3_lib}/_sysconfigdata_*; do
+		f=${f##*/}
+		_PYTHON_SYSCONFIGDATA_NAME=${f%.py}
+	done
+	export _PYTHON_SYSCONFIGDATA_NAME
```

Comment:
```suggestion
	for f in ${XBPS_CROSS_BASE}/${py3_lib}/_sysconfigdata_*; do
		[ -f "$f" ] || continue
		f=${f##*/}
		_PYTHON_SYSCONFIGDATA_NAME=${f%.py}
	done
	[ -n "$_PYTHON_SYSCONFIGDATA_NAME" ] && export _PYTHON_SYSCONFIGDATA_NAME
```
This loop has always kind of bugged me. I've always wondered whether we should break after the first match or continue looping to pick the last match. (In practice, there is probably only one, so it shouldn't matter, but still...)

Without taking a position on that matter, let's at least confirm the file exists to avoid picking up a null match and make sure we picked something before exporting it.

Reply:
fixed that loop.

based on a cursory glance, the numpy and this build helper should not conflict. the only thing they do in common is add to C/LD flags, but both are `+=` so it _should_ be fine